to-be-ccpp-ized/utilities/coords_1d.F90

```diff
@@ -0,0 +1,152 @@
+module coords_1d
```

Comment:
Just confirming the directory is to be named `to-be-ccpp-ized` or `to-be-ccppized` (https://github.com/cacraigucar/atmospheric_physics/tree/atmos_phys_zmcleanup3/to_be_ccppized) since I've seen two variants around!

Reply:
Good catch @jimmielin!

@nusbaume, @cacraigucar, @peverwhee, we might have discussed this previously so my apologies for not taking better notes but do we have a preferred layout/naming convention for these non-ccppized files?  Particularly with coords1d and linear_1d_operators being helper files (which might be needed by other files later) and the high level interface in the solver file.

I think having them at separate layers might be best for organization but not too sure on the naming conventions.

Reply:
After discussing this with @jimmielin and @cacraigucar I believe we decided for the `to_be_ccppized` directory to be completely flat.  This was to essentially make that directory ugly on purpose so that future users/developers remain motivated to get everything CCPP-ized.  Of course if anyone disagrees with this plan please let me know!

@jimmielin @cacraigucar @peverwhee

Reply:
Flattened!